README.md

```diff
@@ -133,8 +131,10 @@ S3 bucket, and if private credentials need adding or updating, they need to be u
 
 You can download and update credentials like this
 
-    aws s3 cp s3://contributions-private/DEV/contributions.private.conf /etc/gu
-    aws s3 cp /etc/gu/contributions.private.conf s3://contributions-private/DEV/
+```
+sudo aws s3 cp s3://contributions-private/DEV/contributions.private.conf /etc/gu/ --profile membership
```

Comment:
I think it's probably better practice to leave the `sudo` out, and let the developer handle the file. Typically there's no need to run `aws` as root if you can manually move the file afterwards (But that's me being paranoid and pedantic, I'll go put my tinfoil hat now if you don't mind)